docs/concepts/service-internals.md

```diff
@@ -102,9 +102,9 @@ Such connection can be used in a typical client-server architecture, that the cl
 <a name="pubsub_client"></a>
 
 ### The PubSub WebSocket client
-The service also supports a specific subprotocol called `json.webpubsub.azure.v1` which empowers the clients to do publish/subscribe directly instead of a round trip to the upstream server. We call the WebSocket connection with `json.webpubsub.azure.v1` subprotocol a PubSub WebSocket client.
+The service also supports a list of specific subprotocol includes `json.webpubsub.azure.v1` and `protobuf.webpubsub.azure.v1` which empowers the clients to do publish/subscribe directly instead of a round trip to the upstream server. We call the WebSocket connection with these subprotocol a PubSub WebSocket client.
```

Comment:
```suggestion
The service also supports a list of specific subprotocols includeing `json.webpubsub.azure.v1` and `protobuf.webpubsub.azure.v1` which empowers the clients to do publish/subscribe directly instead of a round trip to the upstream server. We call the WebSocket connections with these subprotocols PubSub WebSocket clients.
```